projects/ngrx.io/content/guide/migration/v7.md

```diff
@@ -10,6 +10,10 @@ To update your packages to the latest released version, run the command below.
 ng update @ngrx/store
 ```
 
+## Dependencies
+
+NgRx now has a minimum version requirement on Angular 7.
```

Comment:
```suggestion
V7 has the minimum version requirements:

- Angular version 7
- TypeScript 3.1.x
- RxJS 6.x

```

Reply:
Nit, should we add "version" to every dependency or to none of the dependencies?

Reply:
I know this is a point of emphasis on the Angular side. I'd say add to all of them